src/pages/EmergencyNumbers.jsx

```diff
@@ -0,0 +1,121 @@
+import { PhoneIcon } from '@chakra-ui/icons';
+import React, { useEffect, useState } from 'react';
+import { Button, Box, Stack, Text, VStack, Center, Tbody, Th, Tr, Td } from '@chakra-ui/react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Button' is defined but never used.